components/emoji_picker/emoji_picker.jsx

```diff
@@ -585,13 +587,12 @@ export default class EmojiPicker extends React.PureComponent {
         }
         return (
             <div
-                className='emoji-picker'
                 style={pickerStyle}
             >
-                {this.emojiCategories()}
                 {this.emojiSearch()}
+                {this.emojiCategories()}
                 {this.emojiCurrentResults()}
-                <EmojiPickerPreview emoji={this.getCurrentEmojiByCursor(this.state.cursor)}/>
+                <EmojiPickerPreview emoji={this.state.selected}/>
```

Comment:
I don't see this state being set anywhere, so I think this might've been a merge conflict